### PR TITLE
hex_a_hop: Add recipe

### PR DESCRIPTION
**Pull request by win8linux**
*Hex-a-Hop* is a hexagonal tile-based puzzle game wherein the player must eliminate all green tiles by walking on them. Limitless undos, no time limits, just get rid of the green tiles!  Different kinds of tiles are introduced as the player progresses further for variety.

Running on `x86_64`:
screenshot1
screenshot3
screenshot5
screenshot6

### CODE REVIEW

**Comment by win8linux** [comment]
This is the first time I've encountered Autotools, so there may be some unnecessary build steps that need to be removed.

File: games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe
Comment (Begasus) [inline games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe:21]: OK for secondary architecture: 
```diff
-SECONDARY_ARCHITECTURES="!x86"
+SECONDARY_ARCHITECTURES="x86"
```

File: games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe
Comment (Begasus) [inline games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe:63]: Not needed:
```diff
-	mkdir -p $appsDir
```

**Review by Begasus — CHANGES_REQUESTED** [review]

**Review by win8linux — COMMENTED** [review]
All requested changes have been made.

File: games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe
Comment (humdingerb) [inline games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe:11]:
```suggestion
HOMEPAGE="https://hexahop.sourceforge.net"
```

File: games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe
Comment (Begasus) [inline games-puzzle/hex_a_hop/hex_a_hop-1.1.0.recipe:65]: Nitpicking maybe :) (suggestion)
```
	mv $appsDir/hex-a-hop $appsDir/Hex-a-hop
	addAppDeskbarSymlink $appsDir/Hex-a-hop
```

**Review by Begasus — APPROVED** [review]

**Comment by Begasus** [comment]
@humdingerb maybe you could bring your magic skills and create an icon? ;)

**Comment by Begasus** [comment]
LGTM, if only we had an icon, but can be added later. Thanks @win8linux

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
> @humdingerb maybe you could bring your magic skills and create an icon? ;)

I'm not that good an iconizer... we'd need @Handmaus for best results. 😃

**Comment by win8linux** [comment]
Could just make an issue for an icon after the merge.

**Comment by humdingerb** [comment]
> Could just make an issue for an icon after the merge.

Yes, and/or advertise on this thread:
https://discuss.haiku-os.org/t/missing-icons-rdef-hvif-iom

**Comment by win8linux** [comment]
> Yes, and/or advertise on this thread:
> https://discuss.haiku-os.org/t/missing-icons-rdef-hvif-iom

Right, make the issue then post it on that thread. Got it.

**Pull request merged**
